_i18n/en.yml

```diff
@@ -126,12 +126,19 @@ hangouts:
     monero-site: Where the development of this website is coordinated.
 
 merchants:
-  intro1: Merchants of all kinds have come to value the financial privacy that Monero brings. Below is a list of the merchants that we know of that currently accept Monero for their goods and services. <b>These merchants are not vetted by the Getmonero team, see the disclaimer at the bottom of this page</b>. If a company no longer accepts Monero or you would like your business to be listed, please
-  intro2:  open a GitHub issue and let us know
-  intro3: (select the "Merchant" template and fill up all sections).
+  descr: On this page we list non-custodial P2P exchanges and links to merchants directories.
+  useit: "Use Monero!"
+  use: Over the years, merchants of all kinds have come to value the financial privacy that Monero brings. In fact, merchant adoption is continuously rising.
+  dirdescr: "If you are looking for places to spend or exchange your Monero, the directories below have some options. Please remember to do your own research before trusting them with your money:"
+  descrp2p: "We believe it's important to encourage the use of non-centralized exchanges, so we created the next section, which lists non-custodial P2P exchanges and their characteristics. Remember: if you don't hold the keys of your wallet, those coins are not yours."
+  bisqdescr: Decentralized P2P exchange based on Bitcoin and Tor.
+  cardfoss: "Open source:"
+  cardkyc: "<b>No KYC</b> (Know Your Customer):"
+  cardbtcfiat: (BTC &#8596; fiat possible)
+  cardonion: "Onion address:"
+  hodlhodldescr: P2P trading platform with non-custodial escrow.
   disclaimer: |
     "Please note: these links are being provided as a convenience and for informational purposes only; they do not constitute an endorsement by the Monero community of any products, services or opinions of the corporations or organizations or individuals listed. The Monero community bears no responsibility for the accuracy, legality, or content of these external sites. Contact the external site for answers to questions regarding its content. As always, caveat emptor ('buyer beware'); you are responsible for doing your own research. Always use judgement when making online purchases."
```

Comment:
Unrelated but there `" "` show up on the page, is it intentional?

Reply:
I don't know to be honest, but could be.